conf/modules.config

```diff
@@ -281,15 +281,15 @@ if (params.remove_ribo_rna) {
 
 if (!params.skip_alignment) {
     process {
-        withName: 'NFCORE_RNASEQ:RNASEQ:.*:BAM_SORT_SAMTOOLS:BAM_STATS_SAMTOOLS:.*' {
+        withName: '.*:.*:.*:BAM_SORT_SAMTOOLS:BAM_STATS_SAMTOOLS:.*' {
```

Comment:
Why make this more generic?
The `.*` will also match `:` so one really needs to be careful with this. I feel this makes maintainability more troublesome. 
I'll comment on the other important pair lower down.

Reply:
> `The .* will also match : `

Ah, pants! I didn't know this. It's almost safer to just use fully-qualified process selectors everywhere to save making these sorts of innocent mistakes...

Reply:
All reverted in https://github.com/nf-core/rnaseq/pull/766/commits/57792a6ed8cc07710207441284f2aff6b734b085